### PR TITLE
Rewards upd

### PR DESCRIPTION
**Pull request by deesingh08**

### CODE REVIEW

File: crowdloans/projects/polkadot.json [hunk 1]
```diff
@@ -160,7 +160,7 @@
     "overview": "Acala is the decentralized finance network and liquidity hub of Polkadot. It’s a layer-1 smart contract platform that’s scalable, Ethereum-compatible, and optimized for DeFi with built-in liquidity and ready-made financial applications.",
     "website": "https://acala.network/",
     "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/acala.svg",
-    "rewardStatus": "PARALLEL_IS_DISTRIBUTING",
+    "rewardStatus": "PARALLEL_TO_DISTRIBUTE",
```
Comment (Helen8) [inline crowdloans/projects/polkadot.json:163]: For Acala - it should now say "Parallel is distributing"
Production is not correct: https://crowdloan.parallel.fi/#/auction/rewards/polkadot

**Review by Helen8 — COMMENTED** [review]
Added comment for Acala

File: crowdloans/projects/polkadot.json (hunk 1 above)
Comment (deesingh08) [reply crowdloans/projects/polkadot.json:163]: Notion document is updated after I made the changes. Will add this and create a PR

Comment (Helen8) [reply crowdloans/projects/polkadot.json:163]: "Parallel is distributing"

Comment (Helen8) [reply crowdloans/projects/polkadot.json:163]: For Acala

**Review by Helen8 — COMMENTED** [review]
"Parallel is distributing" - Acala status

**Pull request closed without merging**